elements/pfe-sass/maps/_general.scss

```diff
@@ -1,26 +1,48 @@
+@import "@patternfly/patternfly/sass-utilities/scss-variables";
+
 $pfe-vars: (
-  container-spacer:              16px,
-  container-padding:             16px,
-  content-spacer:                24px,
 
-  font-size:                     16px,
-  line-height:                   1.5,
+  container-spacer:              pf-font-prem(16px),
+  container-padding:             pf-font-prem(16px),
+  content-spacer:                pf-font-prem(24px),
+  content-spacer--heading--lg:   pf-font-prem(32px),      
+  content-spacer--heading--md:   pf-font-prem(24px),
+  content-spacer--heading--sm:   pf-font-prem(16px),
+
+  font-family: '"Red Hat Text", "RedHatText", "Overpass", Overpass, Arial, sans-serif',
+  font-family--heading: '"Red Hat Display", "RedHatDisplay", "Overpass", Overpass, Arial, sans-serif',
+  font-family--code: '"Overpass Mono", Consolas, Monaco, "Andale Mono", monospace',
+
+  line-height--heading:     1.2,
+  line-height:              1.3,
 
-  font-family:                   '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--heading:          '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--code:             '"Overpass Mono", Consolas, Monaco, 'Andale Mono', monospace',
+  // General font size variables, used with math to set t-shirt variable values
+  root-font-size:           1em,  // must be EM or PX unit, everything else is rems
```

Comment:
```suggestion
  root--font-size:           1em,  // must be EM or PX unit, everything else is rems
```

Reply:
I changed the var to `font-size--root` to follow the convention of the others